ADD N7 PROFILE PLOTTING FUNCTIONALITY

lucaskearns opened the pull request:
Add --N7profile flag that enables N7 reactivities to be plotted via arcPlot.

Add ability for ReactivityProfiles file to process .txtga extension profiles.

lucaskearns committed:
ADD N7 PROFILE PLOTTING FUNCTIONALITY

Add --N7profile flag that enables N7 reactivities to be plotted
via arcPlot.

Add ability for ReactivityProfiles file to process .txtga extension
profiles.

anthonymustoe reviewed (changes requested):
Can you explain logic of filtering out 'G' in line 331?

Beginngin with line 346, why not combine all the if N7 logic into one block, rather than going back farther to not N7 multiple times? Would make it easier to read. 

Line 535/536 -- can you delete extraneous print statements?

At line 535, this seems like a huge block of code that copies functions from elsewhere. Can you clean this up so there is not copied function?

On line 1300, please remove "import ReactivityProfile" since already imported at top of script.  Also will need to fix line 1301, to profile=ReactivityProfile(N7File)

lucaskearns committed:
CLEAN UP CODE FOR PULL REQUEST

No changes in functionality. Cleaned up code and consolidated / deleted extraneous
lines of code and print statements.

anthonymustoe commented:
Can you remove __pycache__ from the commit. Ready to go after that.

lucaskearns committed:
DELETE __PYCACHE__

Removed generated file for pull request.
